pkgs/development/python-modules/pyctr/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, buildPythonPackage, fetchPypi, pythonAtLeast
+, pycryptodomex }:
+
+buildPythonPackage rec {
+  pname = "pyctr";
+  version = "0.6.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-05lMcsIeJIHI3LwHQTjr4M+bn1FG+GQscuGq34XxjK8=";
+  };
+
+  propagatedBuildInputs = [ pycryptodomex ];
+
+  doCheck = pythonAtLeast "3.6";
```

```suggestion
    disabled = pythonOlder "3.7";
```

see https://github.com/ihaveamac/pyctr/blob/master/setup.py#L12